content-src/experiments/email-tabs.yaml

```diff
@@ -0,0 +1,75 @@
+id: 20
+title: 'Email Tabs'
+slug: email-tabs
+is_featured: false
+platforms: ['addon']
+min_release: 62
+thumbnail: /static/images/experiments/email-tabs/email-tabs.png
+description: >
+  Easily create emails from all of your tabs and make saving and sharing easier than ever.
+introduction: >
+  <p>Ever needed to save or share a whole bunch of tabs as you research, shop, or just browse the web? Email tabs lets you create beautiful emails from your open tabs to save them for later or share them. You can use Email Tabs to automatically send along links, screenshots, or even the text from articles.<p>
+  <p>Email tabs currently works with the Gmail webmail client, but we’re working to bring it to other popular webmail providers as well.</p>
```

Comment:
"Email Tabs" here too (to echo Flodolo's previous comment).

Reply:
Also, I'm not 100% on what this PR means: https://github.com/mozilla/email-tabs/pull/212
But my gut is telling me that once we merge that PR, this comment may be inaccurate (if indeed we'd be supporting Gmail, Yahoo! Mail and Outlook).

Reply:
Yahoo and outlook will be disabled in the ui, there is a switch in `about:addons` options to enable them for qa testing. The plan is to let qa hammer on it and try and get the most common edgecases before we enable them for the txp users.